modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -75,22 +68,33 @@ inline string reverse(string str){
 }
 ----
 
-== Use GitHub to store UDFs
+== Upload UDFs
 
-You can configure GSQL to read from a GitHub repository for ExprFunctions and ExprUtil.
+There are two ways to modify these files to add user-defined functions to GSQL:
 
-If GitHub access is configured, GSQL will retrieve user source code files from GitHub before files added via `PUT`, so long as the files exist.
+* Store UDF files in a GitHub repository, and configure GSQL to read from the repository.
+** This is the recommended approach.
```

Comment:
Where is the security angle to GitHub access?

Reply:
What do you mean by security angle?